build/_npm.shade

```diff
@@ -6,5 +6,10 @@ var npmGloballyInstalled = '${TestCommand("npm" , "--version")}'
 var npmCmd = '${ npmGloballyInstalled ? "npm" : Path.Combine(nodeDir, "npm.cmd") }'
 
 node-install once='installNode'
+
+- // Ensure nodeDir exists. npmCommand is likely a package installation targeting that directory.
+exec program='cmd' commandline='/C mkdir ${nodeDir}' if='!IsLinux && !Directory.Exists(nodeDir)'
```

Comment:
none.  I just mimicked _robocopy-delete.shade rather than, say, _node-install.shade.  in retrospect, _robocopy-delete.shade is the outlier.

I'll switch to use `CreateDirectory()` in next iteration...